project/src/test/java/com/google/sps/data/DataAccessorTest.java

```diff
@@ -162,6 +261,34 @@ public void storeUserPreferences_emptydUserId_throwIllegalArgumentException() {
         () -> dataAccessor.storeUserPreferences("", userPrefs));
   }
 
+  private Entity createRecomEntityByProperties(String userId, String placeId, boolean chosen,
```

Comment:
nit: why not `createRecommendationEntity`?

Reply:
No good reason, changed :)